modules/admin-ui/src/main/webapp/scripts/shared/directives/statsDirective.js

```diff
@@ -58,7 +58,6 @@ angular.module('adminNg.directives')
         stats: '='
       },
       link: function (scope) {
-        scope.stats.fetch();
```

Comment:
Fetching at this point of time doesn't reliable work as the stats service might not yet been configured